services/datcore-adapter/src/simcore_service_datcore_adapter/utils/client_base.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from dataclasses import dataclass
+from typing import Optional, Type
+
+import httpx
+from fastapi import FastAPI
+
+from .app_data import AppDataMixin
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class BaseServiceClientApi(AppDataMixin):
+    """
+    - wrapper around thin-client to simplify service's API calls
+    - sets endspoint upon construction
+    - MIME type: application/json
+    - processes responses, returning data or raising formatted HTTP exception
+    - helpers to create a unique client instance per application and service
+    """
+
+    client: httpx.AsyncClient
+    service_name: str
+    health_check_path: str = "/"
+
+    async def is_responsive(self) -> bool:
```

Comment:
since you are using this in a healthchek API call directly, could you make sure it times out very fast?

Reply:
set to 1 sec